README.rst

```diff
@@ -42,9 +43,10 @@ look at the `examples source`_ for the underlying reStructuredText).
 .. _Sandstone: http://bootswatch.com/sandstone
 .. _Sphinx Bootstrap Theme: http://ryan-roemer.github.com/sphinx-bootstrap-theme
 .. _examples page: http://ryan-roemer.github.com/sphinx-bootstrap-theme/examples.html
-.. _examples source: http://ryan-roemer.github.com/sphinx-bootstrap-theme/_sources/examples.txt
+.. _examples source: http://ryan-roemer.github.com/sphinx-bootstrap-theme/_sources/examples.rst.txt
 .. _Django Cloud Browser: http://ryan-roemer.github.com/django-cloud-browser
-.. _C++ Format: http://cppformat.readthedocs.org
+.. _fmtlib: http://fmtlib.net
```

Comment:
I think this project uses `basic-bootstrap` and not this theme -- https://github.com/fmtlib/fmt/blob/master/doc/conf.py#L107

Can we remove this link?